addons/core/stringtable.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
why?

Reply:
Might editor save file problem, I will restore this.